slapp/transforms/transform_pipeline.py

```diff
@@ -88,6 +90,26 @@ class TransformPipelineSchema(argschema.ArgSchema):
         required=False,
         default="192k",
         description="passed as bitrate to imageio-ffmpeg.write_frames()")
+    font_file = argschema.fields.Str(
+        required=False,
+        default="/allen/aibs/informatics/isaak/fonts/arial.ttf",
+        description=("Path to font file to be used in constructing the "
+                     "videos and images")
+    )
+
+    @mm.pre_load
+    def validate_font_file(self, data, **kwargs):
+        """
+        validate that the font file provided exists in the system, throw
+        an exception if it doesn't.
+        """
+        if 'font_file' in data:
+            if not os.path.exists(data['font_file']):
+                raise TransformPipelineException(
+                    "Provided font file path does not exist. Please "
+                    "enter a valid font file path."
```

Comment:
If you do end up keeping this `pre_load` and I don't  think you need to, you could tell the user what file does not exist. The argschema validation will explicitly tell you "this parameter was suppposed to be a readable file and /blah/blah/file.txt is not readable".

Reply:
Removed the str and made input file!